README.md

```diff
@@ -139,20 +180,32 @@ Response
 
 ```json5
 {
-  "some file id": "some token",
-  "another file id": "some other token"
+  "sessionId": "mySessionId",
+  "files": {
+    "someFileId": "someFileToken",
+    "someOtherFileId": "someOtherFileToken"
+  }
 }
 ```
 
-### 3.2 Send File
+Errors
+
+| HTTP code | Message                    |
+|-----------|----------------------------|
+| 400       | Invalid body               |
+| 403       | Rejected                   |
+| 409       | Blocked by another session |
+| 500       | Unknown error by receiver  |
+ 
+### 4.2 Send File
```

Comment:
https://github.com/localsend/localsend/issues/384

Provides Range requests supports, can allows client-side implement multi-thread sliced download, increase maximum download speed.

see https://developer.mozilla.org/en-US/docs/Web/HTTP/Range_requests

Reply:
https://github.com/localsend/localsend/issues/384

Reply:
Hi, `This route can be called in parallel.` (line 206)

According to your referenced Mozilla page, range requests are mainly used to have a pause/continue feature. I see no effect for transfer speed here as you can already call this route in parallel.

Furthermore, `/send` is used for uploading files (i.e. the sender initiates the file transfer, the receiver is passively accepting bytes)

Range requests are suitable for download which is stated in `5.3 Receive file`.
Here the server may support range requests so the user can partially download the file.
I will update the document soon.